build.rs

```diff
@@ -31,6 +31,7 @@ struct Version {
     major: u32,
     minor: u32,
     patch: u32,
+    stable: bool,
```

Comment:
Could you please add a comment describing why this field exists, and explaining why the version check is correct?